misk-testing/src/main/kotlin/misk/testing/MiskTestExtension.kt

```diff
@@ -19,6 +19,11 @@ internal class MiskTestExtension : BeforeEachCallback, AfterEachCallback {
 
   override fun beforeEach(context: ExtensionContext) {
     Environment.setTesting()
+
+    for (dep in context.getExternalDependencies()) {
```

Comment:
nit: could also extend `BeforeAllCallback` so we don't call startUp for every test (I know it's a no-op).

Reply:
It gets kinda ugly to make this a `BeforeAllCallback`. Currently you do

```kotlin
  @MiskExternalDependency private val dockerSqs = DockerSqs()
```

to be used in a `BeforeAllCallback` it would have to be

```kotlin
companion object {
  @JVMField
  @MiskExternalDependency private val dockerSqs = DockerSqs()
}
```

What do you think?

Reply:
oh gross! yea no need for that